ocs_ci/ocs/perftests.py

```diff
@@ -41,9 +42,46 @@ def setup(self):
         self.benchmark_obj = None  # place holder for the benchmark object
         self.client_pod = None  # Place holder for the client pod object
         self.dev_mode = config.RUN["cli_params"].get("dev_mode")
-        self.environment = get_environment_info()
         self.pod_obj = OCP(kind="pod")
 
+        # Collecting all Environment configuration Software & Hardware
+        # for the performance report.
+        self.environment = get_environment_info()
+        self.environment["clusterID"] = get_running_cluster_id()
+
+        ct_pod = pod.get_ceph_tools_pod()
+        osd_info = ct_pod.exec_ceph_cmd(ceph_cmd="ceph osd df")
```

Comment:
pls consider writing this as a separate function that returns osd related info

Reply:
done